virtio-devices/src/vhost_user/blk.rs

```diff
@@ -261,10 +380,20 @@ impl VirtioDevice for Blk {
                 pause_evt,
                 vu_interrupt_list: interrupt_list_sub,
                 slave_req_handler: None,
+                sockfd: Some(self.vhost_user_blk.as_mut().unwrap().as_raw_fd()),
```

Comment:
Is `as_mut` here really needed? Can you replace it with `as_ref`? This comment applies to other places where `as_raw_fd` is called.

Reply:
I changed them to `as_ref`.